workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/BackButtonScreen.kt

```diff
@@ -28,24 +29,27 @@ public class BackButtonScreen<C : Screen>(
   public val shadow: Boolean = false,
   public val onBackPressed: (() -> Unit)? = null
 ) : Wrapper<Screen, C>, AndroidScreen<BackButtonScreen<C>> {
+  // If they change the shadow value, we need to build a new view to reorder the handlers.
+  override val compatibilityKey: String = keyFor(content, "BackButtonScreen+shadow:$shadow")
+
```

Comment:
nice

Reply:
I suspect you are the only other person on the planet able to appreciate that.